pkg/kubelet/network/cni/cni.go

```diff
@@ -106,10 +106,21 @@ func (plugin *cniNetworkPlugin) SetUpPod(namespace string, name string, id kubec
 	if !ok {
 		return fmt.Errorf("CNI execution called on non-docker runtime")
 	}
-	netns, err := runtime.GetNetNs(id.ContainerID())
+	hostNet, err := runtime.IsUsingHostNetwork(id.ContainerID())
```

Comment:
you can, given the nature of the problem I'd defer the refactor (this is an interface shared with exec plugins)